lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1928,7 +1928,13 @@ class SemanticsObject {
   void dispose() {
     assert(!_isDisposed);
     _isDisposed = true;
-    element.remove();
+
+    EnginePlatformDispatcher.instance.viewManager.safelyBlurElement(
+      element,
+      delayed: false,
+      removeElement: true,
+    );
+
     _parent = null;
     semanticRole?.dispose();
```

Comment:
Some roles will remove their DOM children that may have focus in their implementation of `dispose()`.

Reply:
I didn't see any test break. Is this a behavior that can/will be added later? Do I need to change other classes as well? 

Apologies, I have zero experience with semantics.